static/app/types/organization.tsx

```diff
@@ -273,11 +273,13 @@ export type EventsStats = {
   end?: number;
   isExtrapolatedData?: boolean;
   isMetricsData?: boolean;
+  isMetricsExtractedData?: boolean;
   meta?: {
     fields: Record<string, AggregationOutputType>;
     isMetricsData: boolean;
     tips: {columns?: string; query?: string};
     units: Record<string, string>;
+    isMetricsExtractedData?: boolean;
```

Comment:
Do you expect the value at the top level and also under `meta`?

Reply:
It's sometimes under meta at the top level, sometimes returned in the body (I _think_ that may have been fixed but I'm not sure it's true everywhere), and sometimes in the group-by response meta per-group. 

I don't want it in all these places either, but I'd rather match isMetricsData 1:1 and fix both at the same time in future. It reduces the cognitive load in the meantime so we don't have to remember how one is returned differently.